### PR TITLE
Extend benchmark with consumer group

### PR DESCRIPTION
**Pull request by numinnex**
Extended our benchmark suite with a consumer group poll

### CODE REVIEW

**Review by spetz — APPROVED** [review]
LGTM

**Review by spetz — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -247,6 +247,15 @@ Then, run the benchmarking app with the desired options:
    cargo r --bin iggy-bench -r -- -c -v send-and-poll tcp
    ```
 
+4. Polling with consumer group
+
+   ```bash
+   cargo r --bin iggy-bench -r -- -c -v send --streams 1 --partitions 10 --disable-parallel-producers tcp
+   ```
+   ```bash
```
Comment (hubcio) [inline README.md:256]: missing empty line between 255 and 254

File: bench/src/benchmarks/consumer_group_benchmark.rs [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                stream_id,
+                messages_per_batch,
+                message_batches,
+                warmup_time,
+            );
+            let future = Box::pin(async move { consumer.run().await });
+            futures.as_mut().unwrap().push(future);
+        }
+        info!(
+            "Starting consumer group benchmark {} messages",
```
Comment (hubcio) [inline bench/src/benchmarks/consumer_group_benchmark.rs]: with / for {} messages

File: bench/src/producer.rs [hunk 3]
```diff
@@ -81,6 +88,7 @@ impl Producer {
                 .send_messages(&stream_id, &topic_id, &partitioning, &mut messages)
                 .await?;
         }
+        //println!("stream id: {}, topic id: {}, partitioning: {:?}", stream_id, topic_id, partitioning);
```
Comment (hubcio) [inline bench/src/producer.rs]: remove commented out code

**Review by hubcio — APPROVED** [review]

**Pull request merged**